tests/components/squeezebox/test_config_flow.py

```diff
@@ -166,6 +166,33 @@ async def patch_async_query(self, *args):
     assert result["errors"] == {"base": "invalid_auth"}
 
 
+async def test_form_validate_exception(hass: HomeAssistant) -> None:
+    """Test we handle exception."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": "edit"}
```

Comment:
edit is not a valid source

Reply:
SOURCE_USER then?

Reply:
That sounds more valid, but I think that you need a bit more test to make sure that the rest goes well

Reply:
done

Reply:
actually, no, that doesn't work. This is a validation on a step_id = "edit", so isn't the context correct as "edit". SOURCE_USER aka "user" gives a schema error.  If I call context={"source": "xxx"}, I get an "invalid step_id xxx" error, so isn't the context here the "step_id", which is edit for this step?